src/test/resources/feature/2-mittente/3_downloadFile/103_scaricaAttestazioniOpponibiliNotificaAnnullata.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Mittente visualizza correttamente la notifica in stato Annulato
+
+  @TestSuite
+  @TA_MittenteScaricaAttestazioniDellaNotificaAnnullata
+  @mittente
+  @visualizzazioneNotificheMittente
+
+  Scenario: PN-10245 - Mittente scarica documento AAR della notifica annullata
+    Given PA - Si effettua la login tramite token exchange, e viene visualizzata la dashboard
+    When Nella pagina Piattaforma Notifiche  persona fisica inserire il codice IUN "WVEM-RAVW-HLYV-202405-T-1"
+    And Cliccare sul bottone Filtra
+    And Cliccare sulla notifica restituita
+    And Si visualizza correttamente la section Dettaglio Notifica
+    And Si verifica che la notifica abbia lo stato "Annullata"
+    #And Nella sezione Dettaglio Notifiche si scarica il file Attestazioni Opponibili
```

Comment:
delete this comment

Reply:
this steps are used to download file, which is we are not using now, because Chiara told to just open the file. I have commented them, in order to bring back when we are going to use download method. But i deleted

Reply:
ok, no in this case restore them